build.sbt

```diff
@@ -60,49 +27,12 @@ Global / onLoad := (Global / onLoad).value.andThen { s =>
   s
 }
 
-// this overrides interplay release settings with some adaptations for working with sbt-dynver
-// moreover, it contain only bits necessary for twirl, nothing else
-lazy val releaseSettings: Seq[Setting[_]] = Seq(
-  // Release settings
-  releaseCrossBuild := false,
-  releaseProcess := {
-    import ReleaseTransformations._
-
-    def ifDefinedAndTrue(key: SettingKey[Boolean], step: State => State): State => State = { state =>
-      Project.extract(state).getOpt(key in ThisBuild) match {
-        case Some(true) => step(state)
-        case _          => state
-      }
-    }
-
-    Seq[ReleaseStep](
-      checkSnapshotDependencies,
-      runClean,
-      releaseStepCommandAndRemaining("+test"),
-      releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-      releaseStepCommandAndRemaining("+publishSigned"),
-      releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-      ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-      ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeBundleRelease")),
-      pushChanges
-    )
-  }
-)
-
-val commonSettings = javaCompilerSettings ++ headerSettings ++ Seq(
-  scalaVersion := scala212,
-  crossScalaVersions := Seq(scala212, scala213),
-  scalacOptions ++= scalacCompilerParams,
-)
-
 lazy val twirl = project
   .in(file("."))
-  .enablePlugins(PlayRootProject)
   .settings(
-    commonSettings,
-    releaseSettings,
     crossScalaVersions := Nil, // workaround so + uses project-defined variants
-    mimaFailOnNoPrevious := false
+    mimaFailOnNoPrevious := false,
+    publish / skip := true
```

Comment:
Wait, I think this needs an explicit `scalaVersion := Scala212` now. I think that's part of the workaround mentioned in the comment `// workaround so + uses project-defined variants` above.

Reply:
I mean, it's there in the `Common` plugin ( in `override def globalSettings`) but that's not enabled here.

Reply:
I don't understand why this is needed. I will check if it will work to confirm it.

Reply:
when using `+task` the default value is taken from the root project. Then , the fact that the root project has an empty `crossScalaVersions` also impacts how `+task` behaves.

It's a complete mess of cases and sub-cases. I learnt about all that while working on https://github.com/akka/akka-grpc/pull/907 (which is DRAFT for other reasonS.

Reply:
for the moment, I didn't observed any issue related to that. It just works, expect that `test` fails but because of something else. It's also happening in `master` so not related with this PR.